Main.java

```diff
@@ -109,51 +96,97 @@ static void btnCrudGoBackMethod (){
         crudRunning=false;
     }
     static void btnCrudCreateMethod(){
-
+        String result="Waiting access...";
+        try{
+            result=db1.addProduct(conn); //function passes result onto the string
+        }catch (SQLException ex) {
+            ex.printStackTrace();
+        }
+        crudOutputStream.push(result); //send result to stream for render later
     }
     static void btnCrudReadMethod(){
-
+        String result="Waiting access...";
+        try{
+            result=db1.select(conn); //function passes result onto the string
+        }catch (SQLException ex) {
+            ex.printStackTrace();
+        }
+        crudOutputStream.push(result); //send result to stream for render later
     }
     static void btnCrudUpdateMethod(){
-
+        String result="Waiting access...";
+        try{
+            result=db1.update(conn); //function passes result onto the string
+        }catch (SQLException ex) {
+            ex.printStackTrace();
+        }
+        crudOutputStream.push(result); //send result to stream for render later
     }
     static void btnCrudDeleteMethod (){
-
+        String result="Waiting access...";
+        try{
+            result=db1.delete(conn); //function passes result onto the string
+        }catch (SQLException ex) {
+            ex.printStackTrace();
+        }
+        crudOutputStream.push(result); //send result to stream for render later
     }
-    static void simRender(){
-        if (simulationRunning==true) {
+    static void appRender() {
+
+        txtMainStatusBar.setText(txtMainStatusBarOutput); //update the status bar of main frame
+
+        if (crudRunning == true) {
+            crudRenderTest++;
+            if (crudRenderTest>10) crudRenderTest=0;

```

Comment:
to be consistent with your later formatting changes: crudRenderTest > 10 and crudRenderTest = 0 rather than crudRenderTest>10 etc.

Reply:
Yep, this can be corrected or deleted as required, right now this formatting overflows but I'll fix it soon